oh_staff_ui/classes/ImageFileHandler.py

```diff
@@ -127,3 +129,17 @@ def _get_new_image_dimensions(
         scale_factor = max(current_sizes) / max_size
         new_sizes = tuple(int(dimension / scale_factor) for dimension in current_sizes)
         return new_sizes
+
+    def _make_square_thumbnail(self, thumbnail_image: Image.Image) -> Image.Image:
+        """Create a square thumbnail from a rectangular image, with black padding.
+        To be used after image is resized to thumbnail size.
+
+        Arguments:
+        thumbnail_image: image resized such that the longest side is the thumbnail size.
+        Returns: the square thumbnail.
+        """
+        x, y = thumbnail_image.size
```

Comment:
Please replace `x` and `y` with `width` and `height`.  It doesn't matter for the math and we can infer it from documentation of `_get_new_image_dimensions()`, but making it explicit in the code means we don't have to wonder about it.